arq/connections.py

```diff
@@ -169,24 +171,34 @@ def __init__(
     thus allowing job enqueuing.
     """
     settings = settings or RedisSettings()
-    addr = settings.host, settings.port
+
+    assert not (
+        type(settings.host) is str and settings.sentinel
+    ), "str provided for 'host' but 'sentinel' is true; list of sentinels expected"
+
+    if settings.sentinel:
+        addr = settings.host
+
+        async def pool_factory(*args, **kwargs):
+            client = await aioredis.sentinel.create_sentinel_pool(*args, **kwargs)
+            return client.master_for(settings.sentinel_master)
+
+    else:
+        pool_factory = aioredis.create_redis_pool
```

Comment:
I think this should be `aioredis.pool.create_pool`, that might be why the test is failing.